pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -85,23 +166,22 @@ class Channel:
 
     skew_factor = Instrument.control(
         "SKEW?", "SKEW %eS",
-        """ Channel-tochannel skew factor for the specified channel. Each analog channel can be 
-        adjusted + or -100 ns 
-        for a total of 200 ns difference between channels. You can use the oscilloscope's skew 
-        control to remove 
-        cable-delay errors between channels.
+        """ Channel-tochannel skew factor for the specified channel. Each analog channel can be
+        adjusted + or -100 ns for a total of 200 ns difference between channels. You can use
+        the oscilloscope's skew control to remove cable-delay errors between channels.
         """,
         validator=strict_range,
-        values=[-100, 100]
+        values=[-100, 100],
+        get_process=lambda val: float(re.findall(_MATCH_FLOAT, str(val))[0])
```

Comment:
Can you expand on what this is dealing with? Maybe a protocol test on this could illustrate the different possible message formats this is processing with some examples -- maybe there is an easier way?

Reply:
The problem is that there is a firmware bug in the oscilloscope, so that only for this property the value is returned together with the physical units. So the regular exression is needed to get rid of the final "S" in the string.

I could just remove the last character and cast to float but I deemed the regex approach more robust.

I added a protocol unit test to test and clarify this issue in commit f812262532449770ac57d75a704a357c8dd58471

Reply:
Ha, nice. Might be worth it to add that context as a source code comment to this property. 

Now that I understand what's going on, let me propose an easier fix to this, though. Instead of the `get_process` arg, add a ` preprocess_reply=lambda v: v.rstrip('S')` argument. 
This works because `control`'s kwargs get forwarded to [`values()`](https://pymeasure.readthedocs.io/en/latest/api/instruments/instruments.html#pymeasure.instruments.Instrument.values), so you can use `values`' `preprocess_reply` argument to strip off the "S" before value mapping, and the rest works as expected.
This is not that well documented, I think mainly because it's seldomly needed.

Reply:
Thank you for the suggestion. I applied it in commit fd28778e0e1f8ce9dd4134658def80df5a599846